tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -42,6 +48,8 @@ class SerializableModelLibraryGenerator {
       config: config,
     );
 
+    bool isParentClass = classDefinition.childClasses.isNotEmpty;
```

Comment:
Suggestion: Make this into a getter on the classDefinition instead. Then you do not need to pass this variable all the way down in the call stack and can instead access it by `classDefinition.isParentClass`